models/marts/educ_serv/staging/stg_descr_mat.sql

```diff
@@ -17,10 +17,16 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #}
 with
     raw_data as (
-        select mat.id_eco, mat.mat, mat.descr, mat.descr_abreg
+        select
+            mat.id_eco,
+            mat.mat,
+            mat.descr as description,
+            mat.descr_abreg as description_abreg,
+            row_number() over (partition by mat.mat order by mat.id_eco desc) as seqid
```

Comment:
@sadqim  c'est incohérent avec to test. Tu test pour une resolution par description de compétence mais à la vue de ton code tu resout par mat uniquement. Il faut modifier l'un des deux ! Le test doit tester l'implémentation.

Reply:
true! réglé